### PR TITLE
Landing page of e-com made using react.

### PR DESCRIPTION
**Pull request by j24m**
Hello, created landing page of e-commerce website using react, kindly review.

### CODE REVIEW

**Review by thenorthkun — APPROVED** [review]
UI looks neat, nice use of microanimations. Overall the code is good.

File: public/index.html [hunk 1]
```diff
@@ -9,7 +9,10 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <link rel="stylesheet" href="https://flameui.netlify.app/components.css"/>
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link href="https://fonts.googleapis.com/css2?family=Dancing+Script:wght@400;500;600&family=Roboto+Condensed:ital,wght@0,300;0,400;0,700;1,300;1,400&family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <title>FlameKeys | homepage</title>
     <!--
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
```
Comment (zoef-yt) [inline public/index.html:18]: Remove the commented code.

**Review by zoef-yt — APPROVED** [review]
The code looks good. The UI looks absolutely amazing good work Jyotsna. Good to merge

File: public/index.html (hunk 1 above)
Comment (j24m) [reply public/index.html:18]: Will do.

**Pull request merged**